Contuity/Models/FollowUp.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  FollowUp.swift
+//  Contuity
+//
+//  Created by Lexi Spikerman on 10/2/18.
+//  Copyright © 2018 Generate. All rights reserved.
+//
+
+import Foundation
+
+class FollowUp {
```

Comment:
Models should actually be structs.

The general swift rule is never make a struct into a class unless you need to

Reply:
Gonna use classes because there will be functions for them later on

Reply:
Sorry, I don’t understand what you are getting at.

Structs can still have functions. Models should ideally not be value types, not reference types. Having them be structs leads to less side effects (due to let binding immutability).

Additionally, the goal is that models contain relatively little presentation function logic; that functionality is going to be delegated to presenters and models can be much “dumber” (for lack of a better word).